rocky/assets/css/components/report.scss

```diff
@@ -95,3 +95,15 @@
     }
   }
 }
+
+span {
+  &.label {
+    display:inline-flex;
+    max-width:300px;
+    border-radius:4px;
+    padding:4px 8px;
```

Comment:
Rems?

Reply:
@HeleenSG Do we have vars for these? I couldn't find it.